opm/parser/eclipse/EclipseState/Tables/TableManager.cpp

```diff
@@ -145,6 +146,21 @@ namespace Opm {
         addTables( "RVVD", m_eqldims->getNumEquilRegions());
 
         {
+            size_t numMiscibleTables = ParserKeywords::MISCIBLE::NTMISC::defaultValue;
+            if (deck.hasKeyword<ParserKeywords::MISCIBLE>()) {
+                auto keyword = deck.getKeyword<ParserKeywords::MISCIBLE>();
+                auto record = keyword->getRecord(0);
+                numMiscibleTables =  static_cast<size_t>(record->getItem<ParserKeywords::MISCIBLE::NTMISC>()->getInt(0));
+            }
```

Comment:
I'm glad you are using the `ParserKeywords` classes - I hope it feels reasonably natural!